usaspending_api/api_docs/markdown/data_dictionary.md

```diff
@@ -0,0 +1,750 @@
+| data_source | String | The source of this entry, either Data Broker (DBR) or USASpending (USA) |
+| type | String | 	The mechanism used to distribute funding. The federal government can distribute funding in several forms. These award types include contracts, grants, loans, and direct payments. |
+| type_description | String | The plain text description of the type of the award |
+| piid | String | Procurement Instrument Identifier - A unique identifier assigned to a federal contract, purchase order, basic ordering agreement, basic agreement, and blanket purchase agreement. It is used to track the contract, and any modifications or transactions related to it. After October 2017, it is between 13 and 17 digits, both letters and numbers. |
+| parent_award | Relation | The parent award, if applicable |
+| fain | String | An identification code assigned to each financial assistance award tracking purposes. The FAIN is tied to that award (and all future modifications to that award) throughout the award’s life. Each FAIN is assigned by an agency. Within an agency, FAIN are unique: each new award must be issued a new FAIN. FAIN stands for Federal Award Identification Number, though the digits are letters, not numbers. |
+| uri | String | The uri of the award |
+| total_obligation | Float | The amount of money the government is obligated to pay for the award |
+| total_outlay | Float | The total amount of money paid out for this award |
+| awarding_agency | Relation | The awarding agency for the award |
+| funding_agency | Relation | The funding agency for the award |
+| date_signed | Date | The date the award was signed |
+| recipient | Relation | The recipient of the award |
+| description | String | A description of the award |
+| period_of_performance_start_date | Date | The start date for the period of performance |
+| period_of_performance_current_end_date | Date | The current, not original, period of performance end date |
+| place_of_performance | Relation | The principal place of business, where the majority of the work is performed. For example, in a manufacturing contract, this would be the main plant where items are produced. |
+| potential_total_value_of_award | Float | The sum of the potential_value_of_award from associated transactions |
+| last_modified_date | Date | The date this award was last modified |
+| certified_date | Date | The date this record was certified |
+| create_date | Datetime | The date this record was created in the API |
+| update_date | Datetime | The last time this record was updated in the API |
+| latest_submission | Relation | The submission attribute object that created this award |
+| latest_transaction | Relation | The latest transaction by action_date associated with this award |
+
+#### Transaction Endpoints <a name="transaction"></a>
+
+Transaction data is returned as a combination of the three following models:
+This data is represented internally as the model: `Transaction`
```

Comment:
What do these "internal representation" lines mean? What would an external API user be doing with this information?